iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/ShortBackupRetentionCheck.java

```diff
@@ -62,6 +65,22 @@ protected void registerResourceConsumer() {
         .attribute("retention_in_hours")
           .reportIfAbsent(OMITTING_MESSAGE)
           .reportIf(lessThan(backupRetentionDuration * 24), MESSAGE));
+
+    register("azurerm_app_service",
+      resource -> {
+        BlockSymbol backup = resource.block("backup").reportIfAbsent(OMITTING_MESSAGE);
+        AttributeSymbol enabled = backup.attribute("enabled");
+        if (enabled.isPresent()) {
+          enabled.reportIf(isFalse(), "Make sure disabling backup is safe here.");
+          if (enabled.is(isTrue())) {
+            backup.block("schedule")
+              .reportIfAbsent(String.format(OMITTING_MESSAGE, "schedule.retention_period_in_days"))
```

Comment:
I don't see that specified in [SONARIAC-336](https://jira.sonarsource.com/browse/SONARIAC-336).
Same thing for the next call to `reportIfAbsent`, 2 lines below.